shops/templates/shops/city.html

```diff
@@ -5,9 +5,15 @@
 {% block body %}
     <h1>Cities list:</h1>
     {% if all_cities %}
```

Comment:
From the exercise, bonus part:
"On the list of cities page, put in a link to 'add' a new city"